### ISSUE TITLE
[Meeting 1] Meeting notes are blank

### ISSUE THREAD

**Issue by enzanki-ars** [DarbyComputerClub/Meeting-Notes#1]
As I was unable to attend the meeting, would someone fill out the notes for meeting 1 d305933901c94fe6f843ba9f29229d1b68799274

**Comment by Choco31415** [comment]
Alex, you remember how I already created the meeting notes in Slack? Maybe they're not as through as we would like it, but it describes the meeting fairly well:

The club meeting has now been resolved.
We cleaned up the ideas, goals, and tasks of officers.
We also looked into club curriculum a bit.
We also voted on the club name, and as stated above, 3 out of 5 currently favor Computer Club.
We also ate frozen yogurt. (edited)
Yum!
Vanilla is delicious.
